<launch>
	<node
		pkg="control_logic"
		type="wall_follower"
		name="wall_follower"
	/>
</launch>
